src/idv_service/session/create/identity_profile/advanced/advanced.identity.profile.scheme.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Validation = require('../../../../../yoti_common/validation');
+const AdvancedIdentityProfileSchemeConfig = require('./advanced.identity.profile.scheme.config');
+
+class AdvancedIdentityProfileScheme {
+  /**
+   * @param {string} type
+   * @param {string} objective
+   * @param {string} label
+   * @param {AdvancedIdentityProfileSchemeConfig} [config]
+   */
+  constructor(type, objective, label, config) {
+    Validation.isString(type, 'type');
+    this.type = type;
+    Validation.isString(objective, 'objective', true);
+    if (label) this.objective = objective;
```

Comment:
this line should be: if (objective) this.objective = objective;

Reply:
🦅 👁️ !!!!

Thanks!